model/project_ref.go

```diff
@@ -2032,6 +2032,19 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 		if catcher.HasErrors() {
 			return false, errors.Wrapf(catcher.Resolve(), "validating external links")
 		}
+		var pRef *ProjectRef
+		pRef, err = FindBranchProjectRef(projectId)
+		if err != nil {
+			return false, errors.Wrapf(err, "error getting project")
```

Comment:
Error message should just be "getting project" per our error [style guide](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.sp5f9pecb7lw)